hopsworks-web/yo/app/scripts/controllers/featuregroupViewInfoCtrl.js

```diff
@@ -201,70 +196,136 @@ angular.module('hopsWorksApp')
              * Get custom metadata for feature groups
              */
             self.fetchCustomMetadata = function() {
-                FeaturestoreService.getFeaturegroupCustomMetadata(self.projectId, self.featurestore, self.featuregroup).then(
+                FeaturestoreService.getFeaturegroupCustomMetadata(self.projectId, self.featurestore, self.selectedFeaturegroup).then(
                     function (success) {
                         self.customMetadata = success.data.items;
                     });
             };
 
+            self.toggle = function(selectedFeatureGroup) {
+                if(self.selectedFeaturegroup === null) {
+                    self.tgState = true;
+                } else if ((self.selectedFeaturegroup.id === selectedFeatureGroup.id) && self.tgState === true) {
+                    self.tgState = false;
+                    return;
+                } else {
+                    self.tgState = true;
+                }
+            }
+
             /**
              * Initialization function
              */
-            self.init = function () {
-                self.formatCreated = self.formatDate(self.featuregroup.created)
-                self.pythonCode = self.getPythonCode(self.featuregroup)
-                self.scalaCode = self.getScalaCode(self.featuregroup)
-                self.featuregroupType = ""
-                if(self.featuregroup.featuregroupType === self.onDemandFeaturegroupType){
-                    self.featuregroupType = "ON DEMAND"
+            self.view = function (projectId, projectName, featurestore, featuregroups, settings, toggle) {
+
+                if(toggle) {
```

Comment:
Shouldn't this one be taken care by the method itself?

Reply:
I needed a way of disabling it before, but not anymore

Reply:
It is needed because when you select a new version in the dropdown it also calls this method, however I don't want to trigger a toggle (hiding the sidebar).

Reply:
I could break out the toggle call to be called separately from this function, is that desirable?

Reply:
ok sorry, I didn't see that `toggle` was a parameter of the method. I thought it was `tgState`